frontend/tests/util/isolation.ts

```diff
@@ -94,11 +94,20 @@ export const test = base.extend<CustomTestFixtures, CustomWorkerFixtures>({
 });
 
 const runTobiraCommand = async (tobira: TobiraProcess, args: string[]) => {
-    await new Promise(resolve => {
+    await new Promise((resolve, reject) => {
         args.push("-c");
         args.push(tobira.configPath);
         const p = childProcess.spawn(tobira.binaryPath, args);
-        p.on("close", resolve);
+        p.on("close", code => {
+            if (code === 0) {
+                resolve(undefined);
```

Comment:
Is the `undefined` necessary?

Reply:
yup, at least I get type errors otherwise

Reply:
MEH but I guess it makes sense.